testing/src/main/java/io/stargate/it/http/RestApiv2RowsTest.java

```diff
@@ -597,6 +597,7 @@ public void getRowsWithTimestampQuery() throws IOException {
     assertThat(data.size()).isEqualTo(1);
     assertThat(data.get(0).get("id")).isEqualTo("1");
     assertThat(data.get(0).get("firstName")).isEqualTo("John");
+    assertThat(data.get(0).get("created")).isEqualTo(timestamp);
```

Comment:
And here's the minimal change to verify we get value we expect back. Ideally would have more tests but currently there are no existing tests to augment so this should suffice on short term.